packages/app-mobile/root.tsx

```diff
@@ -359,6 +373,18 @@ const appReducer = (state = appDefaultState, action: any) => {
 			newState.noteSideMenuOptions = action.options;
 			break;
 
+		case 'UPDATE_MOBILE_DATA_WARNING':
```

Comment:
The naming convention is something like "OBJECT_VERB", so the actions are namespaced by the object name. So in your case it should be "MOBILE_DATA_WARNING_UPDATE"